Library/Formula/catimg.rb

```diff
@@ -0,0 +1,20 @@
+class Catimg < Formula
+  desc "Insanely fast image printing in your terminal"
+  homepage "https://github.com/posva/catimg"
+  url "https://github.com/posva/catimg/archive/v2.2.1.tar.gz"
+  sha256 "eb76d3baeb5e7382d8839d7d4351794166c0b6e8d777ffe0087a3401f907e991"
+  head "https://github.com/posva/catimg.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"file.b").write("iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAABKElEQVR4XmP8//8/AyMj4/94BgYGRSBmB2INJMwIlIhHEwPJMwIVNwDFGRnaGf5jVQBURYw44/XrDP/BJuLQwMnJwcDFxYUTM0ZERPxfsYKdAattuJyPZBlLuD47Q70+AwM2V1xnICzOsqECoghkAExDcjIzw3sREYbALi4GUECBCFAAVzaguhSknvE/Q/t/RiKcihIzyF7YsJ6BgRinIocRTH13NzMDy9OnvAzvgaF8EYgZNEGpgAETxDNgBWVAl7Cc/izNoPEZEg7X20kPTMb/19f/Z8SRBjZvZgbHPwsLCwMbGxuYbWCAGpCM1xmuQxISmQHJAgpE5ChEdw2hAGYJ1maGJ1NGIUGSUyRjfPv//xroOZDIjAROvf+v/8cZiMBszshAAAAAxMRgwhvL0coAAAAASUVORK5CYII=")
```

Comment:
We have test fixtures that you can use.  Unless you need the PNG to have specific properties, you can do `system "#{bin}/catimg", test_fixtures("test.png")` here instead.